examples/chunking/app.ddlog

```diff
@@ -18,13 +24,16 @@ word_features(
 
 tag?(word_id bigint) Categorical(13).
 
-function ext_training
-  over rows like words_raw
-  returns rows like words
-  implementation "udf/ext_training.py" handles tsv lines.
-
-words +=
-  ext_training(wid, word, pos, tag) :- words_raw(wid, word, pos, tag).
+words(sent_id, word_id, word, pos, true_tag, tag_id) :-
+  words_raw(sent_id, word_id, word, pos, true_tag),
+  tags(tag_id, tag),
+  if true_tag = "B-UCP" then ""
+  else if true_tag = "I-UCP" then ""
```

Comment:
Not exactly the same, but close. I think there was already a bug in the old chunking example, where words in one sentence sometimes had different sent_ids (likely because there was no row-order enforced):
```
 sent_id | word_id |   word    | pos | true_tag | tag | id
---------+---------+-----------+-----+----------+-----+-----
       5 |     134 | ``        | ``  | O        |   6 | 133
       5 |     135 | The       | DT  | B-NP     |   0 | 134
       5 |     136 | risks     | NNS | I-NP     |   0 | 135
       5 |     137 | for       | IN  | B-PP     |   2 | 136
       5 |     138 | sterling  | NN  | B-NP     |   0 | 137
       5 |     139 | of        | IN  | B-PP     |   2 | 138
       5 |     140 | a         | DT  | B-NP     |   0 | 139
       5 |     141 | bad       | JJ  | I-NP     |   0 | 140
       5 |     142 | trade     | NN  | I-NP     |   0 | 141
       5 |     143 | figure    | NN  | I-NP     |   0 | 142
       5 |     144 | are       | VBP | B-VP     |   1 | 143
       5 |     145 | very      | RB  | B-ADVP   |   4 | 144
       5 |     146 | heavily   | RB  | I-ADVP   |   4 | 145
       5 |     147 | on        | IN  | B-PP     |   2 | 146
       5 |     148 | the       | DT  | B-NP     |   0 | 147
       5 |     149 | down      | JJ  | I-NP     |   0 | 148
       5 |     150 | side      | NN  | I-NP     |   0 | 149
       5 |     151 | ,         | ,   | O        |   6 | 150
       5 |     152 | ''        | ''  | O        |   6 | 151
       5 |     153 | said      | VBD | B-VP     |   1 | 152
       5 |     154 | Chris     | NNP | B-NP     |   0 | 153
       6 |     155 | Dillow    | NNP | I-NP     |   0 | 155
       6 |     156 | ,         | ,   | O        |   6 | 156
       6 |     157 | senior    | JJ  | B-NP     |   0 | 157
       6 |     158 | U.K.      | NNP | I-NP     |   0 | 158
       6 |     159 | economist | NN  | I-NP     |   0 | 159
       6 |     160 | at        | IN  | B-PP     |   2 | 160
       6 |     161 | Nomura    | NNP | B-NP     |   0 | 161
       6 |     162 | Research  | NNP | I-NP     |   0 | 162
       6 |     163 | Institute | NNP | I-NP     |   0 | 163
       6 |     164 | .         | .   | O        |   6 | 164
```